src/domains/transaction/components/SendTransactionForm/SendTransactionForm.tsx

```diff
@@ -67,6 +69,22 @@ export const SendTransactionForm = ({ children, networks, profile, transactionTy
 		history.push(`/profiles/${profile.id()}/wallets/${wallet!.id()}/send-transfer`);
 	};
 
+	const validateFee = () => {
+		if (!isWarning) {
+			setIsWarning(true);
+			setTimeout(() => {
+				const fee = getValues("fee");
+				if (fee?.value) {
+					if (Number(fee.value) < Number(fees.min)) {
+						toasts.warning(t("TRANSACTION.PAGE_TRANSACTION_SEND.VALIDATION.FEE_BELOW_MINIMUM"));
+					}
+
+					setIsWarning(false);
```

Comment:
Validations must be done using the rules from `react-hook-form`.

Reply:
#2951 will introduce some validations

Reply:
As far as I understood this is a warning only, and using `react-hook-form` we gonna prevent the user to proceed with the transaction and this is not the idea, no (https://app.clickup.com/t/8wx5np) ?

Reply:
@luciorubeens I've added a warning variation to the form helper text component since this information would be treated as an error preventing the user to submit the form